### PR TITLE
Fix Kubernetes endpoints SD for empty subsets

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: discovery/kubernetes/endpoints.go [hunk 1]
```diff
@@ -185,13 +185,14 @@ const (
...
 
 func (e *Endpoints) buildEndpoints(eps *apiv1.Endpoints) *targetgroup.Group {
-	if len(eps.Subsets) == 0 {
-		return nil
-	}
-
 	tg := &targetgroup.Group{
 		Source: endpointsSource(eps),
 	}
+	if len(eps.Subsets) == 0 {
```
Comment (brian-brazil) [inline discovery/kubernetes/endpoints.go]: Do you even need this if?

Comment (simonpasquier) [reply discovery/kubernetes/endpoints.go]: Good point, probably not...

**Comment by simonpasquier** [comment]
@brian-brazil PTAL

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**